### ISSUE TITLE
Occasionally the outgoing message is prepended with a character.

### ISSUE THREAD

**Issue by tamaroth** [C1rdec/Poe-Lurker#140]
**Describe the bug**
Sometimes, when copying the buy message from the official trade site, Lurker prepends a character to the entire string and as a result sends it to the person I have previously messaged.
For instance:

```
2020/05/21 22:22:17 515657343 acf [INFO Client 14512] @To HigherMindHeralds: Hi, I would like to buy your Vial of Transcendence listed for 50 chaos in Delirium (stash tab "SHINY!"; position: left 12, top 2)
2020/05/21 22:22:37 515676796 acf [INFO Client 14512] @To HigherMindHeralds: a@StormBrandarchmagedelir Hi, I would like to buy your Vial of Transcendence listed for 50 chaos in Delirium (stash tab "~b/o 50 chaos"; position: left 1, top 16)
2020/05/21 22:22:40 515680656 acf [INFO Client 14512] @To StormBrandarchmagedelir: Hi, I would like to buy your Vial of Transcendence listed for 50 chaos in Delirium (stash tab "~b/o 50 chaos"; position: left 1, top 16)
```

**To Reproduce**
I do not know how to reproduce this. However, when I copied the text from the log file, it was automatically pasted to the game as well. Consider grepping the `send` messages in such a way that they always MUST start with the `@` character to denote private message. Currently it seems it looks anywhere for this.

**Please complete the following information:**
 - Version 1.5.29
 - No log.

**Comment by C1rdec** [comment]
![image](https://user-images.githubusercontent.com/5436436/82614042-de951a00-9b94-11ea-88d2-507a22b756c4.png)

Already done... I'm experience your issue.

**Comment by C1rdec** [comment]
I think I found the issue.

https://github.com/C1rdec/Poe-Lurker/blob/2f238036754ff457dc778c00d9ccde32efcfb4cb/src/Lurker/Helpers/KeyboardHelper.cs#L73

I'll add a delay.

**Comment by C1rdec** [comment]
Should be fix in 1.5.30.
